INDEX

```diff
@@ -401,6 +401,14 @@
 		"official": false,
 		"primary_pkg": null
 	},
+	"photoprism": {
+		"MANIFEST": "photoprism.json",
+		"description": "PhotoPrism® is an AI-Powered Photos App for the Decentralized Web. It makes use of the latest technologies to tag and find pictures automatically without getting in your way.",
+		"icon": "https://icons.freenas.org/community-icons/photoprism.png",
+		"name": "PhotoPrism®",
```

Comment:
Why the discrepancy in the name defined in `photoprism.json` and here ?

Reply:
Didn't realise there's a connection. Updated it now.